routes/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (express) {
   router.use('/user', require('./users')(express))
   router.use('/post', require('./posts')(express))
   router.use('/feed', require('./feed')(express))
+  router.use('/search', require('./search')(express))
```

Comment:
I was thinking of just adding optional query to the `/api/users` endpoint. So I could order results by a desired field. This would be really useful for me.